samples/README.md

```diff
@@ -0,0 +1,4 @@
+Samples migrated
```

Comment:
Please move this into `samples/snippets/README.md`. There are still samples in the general `samples/` directory.